### PR TITLE
MAN2-2876 ZXCVBN bundling

### PR DESCRIPTION
**Pull request by rjstires**
## Purpose
Matching changes for MAN2-2876

### CODE REVIEW

File: src/forms/PasswordInput.js [hunk 1]
```diff
@@ -47,6 +57,10 @@ PasswordInput.propTypes = {
   disabled: PropTypes.bool,
   id: PropTypes.string,
   className: PropTypes.string,
+  strength: PropTypes.oneOfType([
+    PropTypes.number,
+    PropTypes.bool,
```
Comment (asauber) [inline src/forms/PasswordInput.js]: Why not rely on `null` instead of `false`? Does zxcvbn return `false` with empty input?

Comment (asauber) [reply src/forms/PasswordInput.js]: Can simplify this to `PropTypes.number`

File: src/forms/PasswordInput.js [hunk 2]
```diff
@@ -1,38 +1,48 @@
...
+export default function PasswordInput({
+  name,
+  className,
+  disabled,
+  onChange,
+  id,
+  strength,
+  value,
+}) {
+  const showStrength = !isEmpty(value) && isNumber(strength);
```
Comment (asauber) [inline src/forms/PasswordInput.js]: With `PropTypes.number`, `isNumber` can become `!isNull`

File: src/forms/PasswordInput.js (hunk 1 above)
Comment (rjstires) [reply src/forms/PasswordInput.js]: Ill have to pull it up, again, it's been two weeks so I forget at this point.

**Comment by asauber** [comment]
Let's merge this, cut a release, and update manager.

https://github.com/linode/manager/blob/develop/package.json#L91

**Pull request merged**